### PR TITLE
Setup NigeriaLII

### PR DESCRIPTION
**Pull request by nickmwangemi**
image
image
image

### CODE REVIEW

**Comment by niiroobiro** [comment]
@nickmwangemi Please add Gazettes as a menu item. We have gazettes for Nigeria.

**Comment by nickmwangemi** [comment]
> @nickmwangemi Please add Gazettes as a menu item. We have gazettes for Nigeria.

Done

File: nigerialii/templates/peachjam/_footer.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{% extends "peachjam/_footer.html" %}
+{% load i18n static %}
+{% block about %}{% endblock %}
+{% block second-content-column %}
+  <h5>{% trans 'Our partners' %}</h5>
+  <div>
+    <a href="https://agp.africanlii.org/" target="_blank" rel="noreferrer">AfricanLII</a>
```
Comment (longhotsummer) [inline nigerialii/templates/peachjam/_footer.html:7]: drop the agp.

File: nigerialii/templates/peachjam/_footer.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <a href="https://laws.africa/" target="_blank" rel="noreferrer">Laws.Africa</a>
+  </div>
+  <div>
+    <a href="" target="_blank" rel="noreferrer">{% trans '' %}</a>
+  </div>
+{% endblock %}
+{% block footer-logos %}
+  <div class="footer-logos">
+    <a class="footer-logos__item"
+       href="https://agp.africanlii.org/"
```
Comment (longhotsummer) [inline nigerialii/templates/peachjam/_footer.html:19]: drop the agp.

File: nigerialii/templates/liiweb/home.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
+{% extends "liiweb/home.html" %}
+{% load static i18n %}
+{% block title %}
```
Comment (longhotsummer) [inline nigerialii/templates/liiweb/home.html:3]: Identical to the base template, this block can be removed

**Review by longhotsummer — APPROVED** [review]
I'm happy for this to go ahead. We can continue to add new items and hone details once we have it running.

**Pull request merged**